docs-markdown/src/helper/utility.ts

```diff
@@ -525,7 +527,7 @@ export function findReplacement(
 	if (result !== null && result.length) {
 		const groups = !isRegExp(expression) ? expression.groups : null;
 		const match = groups && result.groups ? result.groups[groups[0]] : result[0];
-		if (match) {
+		if (match && match !== value) {
```

Comment:
This is unrelated to my changes, just something I noticed.  There's no need to return a replacement if it's already the value we want to set.  This saves some branches of code from executing when someone attempts to update the ms.date when it's already the current date.  Just a nit pick fix